resources/function_help/json/regexp_match

```diff
@@ -2,9 +2,9 @@
   "name": "regexp_match",
   "type": "function",
   "groups": ["Conditionals", "String"],
-  "description": "Return the first matching position matching a regular expression within a string, or 0 if the substring is not found.",
+  "description": "Return the first matching position matching a regular expression (QRegularExpression) within an unicode string, or 0 if the substring is not found.",
```

Comment:
```suggestion
  "description": "Return the first matching position matching a regular expression within an unicode string, or 0 if the substring is not found.",
```

That's an internal detail only -- I don't think we should expose it here to users.